src/stravalib/client.py

```diff
@@ -1208,7 +1210,7 @@ def get_activity_laps(
             result_fetcher=result_fetcher,
         )
 
-    def get_gear(self, gear_id: str) -> model.Gear:
+    def get_gear(self, gear_id: str) -> strava_model.DetailedGear:
```

Comment:
this is minor  - there are a few other things that get returned / undocumented specifically for shoes that could be useful (but id on't know what users want - 
<img width="504" alt="Screenshot 2024-06-15 at 7 42 56 PM" src="https://github.com/stravalib/stravalib/assets/7649194/7bdeea2f-48a8-4b6f-96fc-2eceaff5b2b5">

retired: bool
primary: bool 

are the two that stand out to me that as a runner i'd track if  when i was running big miles. i think this is optional however and we could also just wait and see if we get user requests for this as i know you also (and i've been doing the same) removed empty classes if they just bring in the same object from strava_model. so please do with this comment as you please!

Reply:
I agree these two seem useful, but choose to add them whenever someone wants to use them :-)